### DISCUSSION TITLE
How load Firebase ANE in Worker?

### DISCUSSION THREAD

**Discussion by JuanManuelCardenas** [airsdk/Adobe-Runtime-Support#2607] [Q&A]
I try to load FIrebase ANE in a Worker but throw me "Error #3731: This feature is not available within this context" Error.

**Comment by teotigraphix** [comment]
Don't think you can do this as they are separate processes and thus memory. 

This sounds familiar to something @ajwfrost was talking about in relation to ANEs and workers.

**Comment by JuanManuelCardenas** [comment]
Is there any progress on this? @ajwfrost
